media/rvva-openapi.json

```diff
@@ -0,0 +1,305 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Processing Actitivities Register API v0.1"
+  },
+  "paths": {
+    "/processing_activities/{processing_activity_id}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/processing_activity_id"
+        }
+      ],
+      "get": {
+        "responses": {
+          "200": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProcessingActivity"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/processing_activities/{processing_activity_id}/versions/{processing_activity_version_number}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/processing_activity_id"
+        },
+        {
+          "$ref": "#/components/parameters/processing_activity_version_number"
+        }
+      ],
+      "get": {
+        "responses": {
+          "200": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProcessingActivityVersion"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/entities/{entity_id}": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/entity_id"
+        }
+      ],
+      "get": {
+        "responses": {
+          "200": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Entity"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "ProcessingActivity": {
+        "properties": {
+          "active": {
+            "description": "If the latest version is still valid",
+            "type": "boolean"
+          },
+          "latest_version": {
+            "description": "The latest valid version of ProcessingActivityVersion",
+            "type": "number"
+          },
+          "processing_activity_id": {

```

Comment:
I would just call this `id` instead of `processing_activity_id.
Otherwise it feels like a stutter: ProcessingActivity.processing_activity_id